_includes/jobrows.html

```diff
@@ -1,16 +1,14 @@
 {% capture jobs_data %}
   {% for job in include.sorted_jobs %}
-  {% capture nowunix %}{{'now' | date: '%m/%d/%Y' | date: %s }}{% endcapture %}
-  {% capture posted %}{{ job.posted | date: "%m/%d/%Y" | date: '%b %d, %Y'}}{% endcapture %}
-  {% capture posted_order %}{{ job.posted | date: "%m/%d/%Y" | date: '%Y%m%d'}}{% endcapture %}
-  {% capture expires_order %}{{ job.expires | date: "%m/%d/%Y" | date: '%Y%m%d'}}{% endcapture %}
-    <tr class='tr odd {% cycle "odd" "even" %}' data-posted="{{ posted_order }}" data-expires="{{ expires_order }}">
+  {% capture posted %}{{ job.posted | date: '%b %d, %Y'}}{% endcapture %}
+  {% capture expires %}{{ job.expires | date: '%b %d, %Y' }}{% endcapture %}
+    <tr class='tr odd {% cycle "odd" "even" %}' data-posted="{{ job.posted }}" data-expires="{{ job.expires }}">
```

Comment:
Should this be just `posted` and `expires` on line 5?

Reply:
`posted` and `expired` are the "pretty printed" dates. The form fields should be in the ISO format because it's the field used in the `$('#min').on('change'...)` event to do the sorting.

As an aside, it would be nice to apply localization for the pretty printed dates, but I couldn't figure out how to do it. A quick search said it could be tricky with Jekyll, so I didn't investigate much further.

Reply:
Ah gotcha, I see where they are used now (it was not changed in this PR and I don't remember my own code):

![image](https://github.com/user-attachments/assets/d8785631-2996-48eb-a1cf-f3289492cb3c)

This looks good then - I'll do a preview and final review a bit later today - going outside to play in the snow for a bit! :snowman: